src/crimson/os/seastore/cache.h

```diff
@@ -448,6 +526,12 @@ class Cache {
     }
   }
 
+  backref_buffer_ref backref_buffer;
+  std::list<backref_buffer_ref> backref_bufs_to_flush;
```

Comment:
It is still not very clear to me why need to cut the backref_buffer into segments?

Why a single large map cannot work?

Reply:
Ah, large map should work, I'll improve that part.

Reply:
Hmm, not sure. The large map orderred by `journal_seq_t` should still be efficient to find out the segment boundaries in order to pass the submap iterators to the "batch insertion" process, right? Or am I still missing something?

From the current code, cutting backref_buffer seems overcomplicated to me in the according `get_oldest_backref_dirty_from()`, `roll_backref_buffer()`, `trim_backref_bufs()`, `batch_insert_from_cache()` logics.

Reply:
Yes, will drop it.